src/compiler/crystal/syntax/parser.cr

```diff
@@ -70,6 +70,20 @@ module Crystal
       #
       # then this flag is set to `true` when parsing `foo`'s arguments.
       @stop_on_do = false
+
+      # This flag tells the parser to stop on comma while parsing call
+      # when there is an ambiguous suffix `+` or `-`
+      # ```plain
```

Comment:
```suggestion
      #
      # ```
```

Reply:
This is intentional. Without language tag the formatter would apply and render the example useless.